apps/lockfile-explorer-web/src/containers/LockfileEntryDetailsView/index.tsx

```diff
@@ -0,0 +1,200 @@
+import React, { useCallback, useEffect, useState } from 'react';
+import styles from './styles.scss';
+import appStyles from '../../appstyles.scss';
+import { IDependencyType, LockfileDependency } from '../../parsing/LockfileDependency';
+import { useAppDispatch, useAppSelector } from '../../store/hooks';
+import { pushToStack, selectCurrentEntry } from '../../store/slices/entrySlice';
+import { ReactNull } from '../../types/ReactNull';
+import { LockfileEntry } from '../../parsing/LockfileEntry';
+
+enum InfluencerTypes {
+  DETERMINANT,
+  TRANSITIVE_REFERRER
```

Comment:
Rush's capitialization convention would be `InfluencerTypes.TransitiveReferrer`.

Reply:
changed